norlab_icp_mapper/Map.cpp

```diff
@@ -739,7 +574,7 @@ norlab_icp_mapper::Map::PM::DataPoints norlab_icp_mapper::Map::getGlobalPointClo
 
 void norlab_icp_mapper::Map::setGlobalPointCloud(const PM::DataPoints& newLocalPointCloud)
 {
-	if(computeProbDynamic && !newLocalPointCloud.descriptorExists("normals"))
+	if(computesDynamicPoints() && !newLocalPointCloud.descriptorExists("normals"))
 	{
 		throw std::runtime_error("compute prob dynamic is set to true, but field normals does not exist for map points.");
```

Comment:
I think we can remove this exception from here, since it is now just a module. We just need to make sure that a clear exception is thrown in the ComputeDynamicsMapperModule class.